recipes/dask-searchcv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dask-searchcv" %}
+{% set version = "0.0.1" %}
+{% set md5 = "7d8fb38ba01d7518e2d22e5933f1e419" %}
```

Comment:
Yeah, we prefer not to use `md5` if possible as that checksum is pretty broken. Given warehouse provides `sha256`, we have encouraged people to use that instead.